lib/sumsub/request.rb

```diff
@@ -151,6 +151,17 @@ def get_access_token(user_id, level_name, ttl_in_seconds: nil, external_action_i
       parse_response(response)
     end
 
+    # API docs: https://developers.sumsub.com/api-reference/#testing-on-the-test-environment
```

Comment:
API docs: https://developers.sumsub.com/api-reference/#testing-on-the-test-environment